compiler/src/dmd/nogc.d

```diff
@@ -13,11 +13,14 @@
 
 module dmd.nogc;
 
+import core.stdc.stdio;
+
 import dmd.aggregate;
 import dmd.apply;
 import dmd.astenums;
 import dmd.declaration;
 import dmd.dscope;
+import dmd.errors;
```

Comment:
Adding these 2 imports seems to be an unrelated change.

Reply:
people keep removing core.stdc.stdio, and I keep adding it back in, because it is for the debugging printf's.